README.md

```diff
@@ -69,19 +98,244 @@ Each endpoint will then describe its method and path.
     :path: "/doc/resource/*"
 ```
 
-#### Special considerations for request bodies with timestamps
+### Quick Start
+
+To quickly implement Betamocks in your application:
```

Comment:
I don't LOVE the word "quickly" here. When is anything ever quick and simple?? 😩 😆 but 100% non-blocking.

Reply:
for real tho